mk/platform.mk

```diff
@@ -93,14 +93,14 @@ endif
 AR := ar
 
 define SET_FROM_CFG
+  $(1) ?=
```

Comment:
How come? This is not different from the default behaviour, assuming that $(1) is used somewhere.